src/sources/data_source.js

```diff
@@ -159,7 +164,10 @@ export default class DataSource {
         }
 
         // # of zoom levels bigger than 256px tiles - 8 in place of log2(256)
-        this.zoom_bias = Math.log2(this.tile_size) - 8;
+        // Many Tangram functions assume 256px tiles, this factor adjusts for the
+        // case of bigger tile sizes - eg 512px tiles are 1 zoom level bigger,
+        // 1024px tiles are 2 levels bigger
+        this.zoom_bias = Math.log2(this.tile_size) - 8 + this.tile_adjust;
```

Comment:
`tile_adjust` should be `zoom_offset`?

Reply:
yeah I just noticed that too! not sure how that got in there…